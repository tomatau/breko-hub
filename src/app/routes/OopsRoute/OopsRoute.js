import React from 'react'
import { Helmet as DocumentMeta } from 'react-helmet-async'
import { oopsRoute as oopsRouteCopy } from 'app/copy'

export default class OopsRoute extends React.Component {
  render() {
    return (
      <>
        <DocumentMeta>
          <title>{oopsRouteCopy.documentTitle}</title>
        </DocumentMeta>
        <section className='OopsRoute'>
          {oopsRouteCopy.content}
        </section>
      </>
    )
  }
}
