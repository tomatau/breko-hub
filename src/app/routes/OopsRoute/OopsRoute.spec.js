import React from 'react'
import OopsRoute from './OopsRoute'

describe(`OopsRoute Component`, function () {
  helpers.setupSnapshots(__filename)

  it(`matches snapshot`, () => {
    expect(
      snap(shallow(<OopsRoute />))
    ).to.matchSnapshot()
  })
})
