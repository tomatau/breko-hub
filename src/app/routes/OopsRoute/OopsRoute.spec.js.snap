// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OopsRoute Component matches snapshot 1`] = `
<Fragment>
  <Helmet
    defer={false}
    encodeSpecialCharacters={true}
  >
    <title>
      Oopsies
    </title>
  </Helmet>
  <section
    className="OopsRoute"
  >
    <h2>
      There was a problem
    </h2>
    <p>
      Something went oops, oh no!
    </p>
    <p>
      We need to fix it, but this is an example.
    </p>
    <p>
      Your developers and managers should change this page :)
    </p>
  </section>
</Fragment>
`;
